addons/product/views/product_views.xml

```diff
@@ -80,6 +80,63 @@
                                                 <field name="date_end"/>
                                                 <field name="applied_on" invisible="1"/>
                                             </tree>
+                                            <kanban string="Pricelist Items" class="o_kanban_mobile" draggable="0" quick_create="false" create="true">
+                                               <field name="pricelist_id" string="Pricelist"/>
+                                               <field name="fixed_price" string="Price"/>
+                                               <field name="min_quantity"/>
+                                               <field name="date_start"/>
+                                               <field name="date_end"/>
+                                               <field name="applied_on" invisible="1"/>
+                                               <field name="price" invisible="1"/>
+                                               <templates>
+                                                   <t t-name="kanban-box">
+                                                       <div t-attf-class="oe_kanban_global_click">
+                                                           <div class="row">
+                                                               <div class="col-12">
+                                                                   <strong>
+                                                                       <t t-esc='record.pricelist_id.string'/>: <field name='pricelist_id'/>
+                                                                   </strong>
+                                                               </div>
+                                                           </div>
+                                                           <div class="row">
+                                                               <div class="col-12">
+                                                                   <span><t t-esc='record.price.string'/>: <t t-esc='record.fixed_price.value'/></span>
+                                                               </div>
+                                                           </div>
+                                                           <div class="row">
+                                                               <div class="col-12 text-muted">
```

Comment:
use either `bold` or normal, no real need to use `muted` (which is usually used for canceled objects).